packages/muelu/src/Interface/MueLu_ParameterListInterpreter_def.hpp

```diff
@@ -971,7 +972,7 @@ namespace MueLu {
 
     // Aggregation scheme
     MUELU_SET_VAR_2LIST(paramList, defaultList, "aggregation: type", std::string, aggType);
-    TEUCHOS_TEST_FOR_EXCEPTION(!strings({"uncoupled", "coupled", "brick", "matlab"}).count(aggType),
+    TEUCHOS_TEST_FOR_EXCEPTION(!strings({"uncoupled", "coupled", "brick", "matlab","notay"}).count(aggType),
```

Comment:
"pairwise" might be more descriptive, my 2 cents.

Reply:
"Notay"  equals the algorithm as described in the paper (or as close as we could come)

"Pairwise" (when we have it)  will describe "our fire-breathing custom pairwise algorithm"